index.js

```diff
@@ -286,6 +286,8 @@ class WebTorrent extends EventEmitter {
       for (const t of this.torrents) {
         if (t.infoHash === torrent.infoHash && t !== torrent) {
           torrent._destroy(new Error(`Cannot add duplicate torrent ${torrent.infoHash}`))
+          ontorrent(t)
+          this.emit('torrent', t)
```

Comment:
I don't agree on emitting torrent again. The callback does make sense.

Similar to https://github.com/webtorrent/webtorrent/pull/2239/files

Reply:
fixed